### ISSUE TITLE
de.js translations missing again in v3.0 (see patch)

### ISSUE THREAD

**Issue by jmiserez** [froala/wysiwyg-editor#3758]
The following translations are missing in de.js and should be added again. They were already added in 2.9 (they are in 2.9.5) in https://github.com/froala/wysiwyg-editor/issues/3225 but are now missing in 3.0.5.
```
'Default': 'Standard',

// Ordered lists
'Lower Alpha': 'Kleinbuchstaben',
'Lower Greek': 'Griechisches Alphabet',
'Lower Roman': 'Römische Ziffern (klein)',
'Upper Alpha': 'Grossbuchstaben',
'Upper Roman': 'Römische Ziffern (gross)',

// Unordered lists
'Circle': 'Kreis',
'Disc': 'Kreis gefüllt',
'Square': 'Quadrat',

// Line height
'Line Height': 'Zeilenhöhe',
'Single': 'Einfach',
'Double': 'Doppelt',
```
The following additional translations should also be added to de.js:
```
'Text Color': 'Textfarbe',
'Background Color': 'Hintergrundfarbe',
```

Related: https://github.com/froala/wysiwyg-editor/issues/3723.

##### Expected behavior.
Translated text (see above)

##### Actual behavior.
Untranslated text.

##### Editor version.
3.0.5

**Comment by jmiserez** [comment]
I have updated the issue and added all missing translations from https://github.com/froala/wysiwyg-editor/issues/3225.

**Comment by bsmarsh333** [comment]
This is still an issue in 3.1.

**Comment by cguglielmo** [comment]
I noticed that version 4.0.11 contains the translations, so they have been added again, at least for v4. Thank you!

**Comment by jmiserez** [comment]
Seems fixed in 4.0.11, closing.
